GithubApp/Controllers/FilterViewController.swift

```diff
@@ -9,27 +9,90 @@ import UIKit
 
 class FilterViewController: UIViewController {
 
+    //MARK: Instance Properties
+    var delegate: FilterDelegate?
+    var filterData :[Filter] = []
+    let checkedImage = UIImage(named: "check")! as UIImage
+    let uncheckedImage = UIImage(named: "uncheck")! as UIImage
+
+    // MARK: IB outlet
     @IBOutlet var filterView: FilterView!
+    
     override func viewDidLoad() {
-        super.viewDidLoad()
+        //super.viewDidLoad()
+        print("View Did load 2 called")
         filterView.filterTableView.delegate = self
         filterView.filterTableView.dataSource = self
     }
-    let languagesData = ["Python","Interior Painting","Electrician","Inappropriate client behavior","Gardener","Carpenter","House Cleaning"]
+    
+    // MARK: Functions
+    func configure(_ filterData: [Filter]){
```

Comment:
ambiguous name. the method is configuring what?

Reply:
name Changed